src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -477,27 +477,16 @@ export type SentryAppInstallation = {
   code?: string;
 };
 
-export type SentryAppWebhookError = {
-  webhookUrl: string;
-  app: {
-    uuid: string;
-    slug: string;
-    name: string;
-  };
-  request: {
-    body: object;
-    headers: object;
-  };
-  eventType: string;
+export type SentryAppWebhookRequest = {
+  webhook_url: string;
```

Comment:
@nolachen We should keep the interfaces in camelCase whenever possible. We have some utility functions that might make it easier such as `to_camel_case_dict`. Using a serializer might make sense as well.